### PR TITLE
enhancements on scicrunch classifiers (frontend)

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?
- [x] Show error msg coming from backend when adding new classifier fails
- [x] Use key for building tree
- [x] Reload classifiers after being successfully added
- [x] Remove ```RRID:``` prefix if the user provides it

### CODE REVIEW

**Review by sanderegg — APPROVED** [review]
:+1:

File: services/web/client/source/class/osparc/dashboard/ClassifiersEditor.js [hunk 1]
```diff
@@ -112,8 +113,7 @@ qx.Class.define("osparc.dashboard.ClassifiersEditor", {
           osparc.store.Store.getInstance().getAllClassifiers(true);
         })
         .catch(err => {
-          osparc.component.message.FlashMessenger.getInstance().logAs(this.tr("Invalid RRID"), "ERROR");
-          console.error(err);
+          osparc.component.message.FlashMessenger.getInstance().logAs(err, "ERROR");
```
Comment (pcrespov) [inline services/web/client/source/class/osparc/dashboard/ClassifiersEditor.js:116]: good!

File: services/web/client/source/class/osparc/dashboard/ClassifiersEditor.js [hunk 2]
```diff
@@ -100,6 +100,7 @@ qx.Class.define("osparc.dashboard.ClassifiersEditor", {
     },
 
     __addRRIDClassfier: function(rrid, btn) {
+      rrid = rrid.replace("RRID:", "");
```
Comment (pcrespov) [inline services/web/client/source/class/osparc/dashboard/ClassifiersEditor.js:103]: perfect

**Review by pcrespov — APPROVED** [review]
🎄🎄🎄🎄

**Review by KZzizzle — APPROVED** [review]
amazing

**Pull request merged**